README.md

```diff
@@ -329,7 +329,24 @@ cargo fmt --all
 
 For non-consensus breaking releases, this project uses the following release process:
 
-1. A release manager enumerates the PRs or issues that would _block_
+1. The release must be timed so that it does not interfere with a *prepare
+phase*.  The timing of the next Stacking cycle can be found
+[here](https://stacking.club/cycles/next). A release to `mainnet` should happen
+at least 24 hours before the start of a new cycle, to avoid interfering
```

Comment:
Does anyone want to add anything to determining how to avoid the *prepare phase*? @kantai ?

Reply:
This looks good to me. Another possible source is https://blockstack.github.io/when-rewards/